smk-config.json

```diff
@@ -101,7 +101,7 @@
+            "popupTemplate": "<div class=\"smk-body\">\n    <div class=\"smk-feature\">\n      \n        <div class=\"smk-layer\">\n            Municipal Regional District Tax: 2%\n        </div>\n        <div class=\"smk-attributes\">\n            <div class=\"smk-attribute\">\n                <span class=\"smk-attribute-title\">\n                    Recipient:\n                </span>\n                <span>\n                    <span class=\"smk-value\">\n                        {{feature.properties.Recipient}}\n                    </span>\n                </span>\n            </div>\n            <div class=\"smk-attribute\">\n                <span class=\"smk-attribute-title\">\n                    Tax Rate:\n                </span>\n                <span>\n                    <span class=\"smk-value\">\n                        {{feature.properties.Tax_Rate}}\n                    </span>\n                </span>\n            </div>\n            <div class=\"smk-attribute\">\n                <div class=\"smk-attribute\">\n                    <span class=\"smk-attribute-title\">\n                        Area:\n                    </span>\n                    <span>\n                        <span class=\"smk-value\">\n                            {{feature.properties.Accom_Area}}\n                        </span>\n                    </span>\n                </div>\n            </div>\n            <div class=\"smk-attribute\">\n                <div class=\"smk-attribute\">\n                    <span class=\"smk-attribute-title\">\n                        Start date:\n                    </span>\n                    <span>\n                        <span class=\"smk-value\">\n                            {{feature.properties.MRDT_Start_Date}}\n                        </span>\n                    </span>\n                </div>\n            </div>\n         \n        </div>\n    </div>\n</div>"
```

Comment:
It's really trivial, but there's still a newline left behind after removing the comments, leaving an extra space:

<div class="smk-body">
    <div class="smk-feature">
      
        <div class="smk-layer">
            Municipal Regional District Tax: 3%
        </div>
        <div class="smk-attributes">
            <div class="smk-attribute">
                <span class="smk-attribute-title">
                    Recipient:
                </span>
                <span>
                    <span class="smk-value">
                        {{feature.properties.Recipient}}
                    </span>
                </span>
            </div>
            <div class="smk-attribute">
                <span class="smk-attribute-title">
                    Tax Rate:
                </span>
                <span>
                    <span class="smk-value">
                        {{feature.properties.Tax_Rate}}
                    </span>
                </span>
            </div>
            <div class="smk-attribute">
                <div class="smk-attribute">
                    <span class="smk-attribute-title">
                        Area:
                    </span>
                    <span>
                        <span class="smk-value">
                            {{feature.properties.Accom_Area}}
                        </span>
                    </span>
                </div>
            </div>
            <div class="smk-attribute">
                <div class="smk-attribute">
                    <span class="smk-attribute-title">
                        Start date:
                    </span>
                    <span>
                        <span class="smk-value">
                            {{feature.properties.MRDT_Start_Date}}
                        </span>
                    </span>
                </div>
            </div>
         
        </div>
    </div>
</div>